ui/src/components/inputs/EditorView.vue

```diff
@@ -179,6 +179,12 @@
                 localStorage.removeItem(restoredLocalStorageKey);
             }
         }
+
+        // validate flow on first load
+        store.dispatch("flow/validateFlow", {flow: flowYaml.value})
```

Comment:
I don't think it's a good idea, the flow must return the error directly from the ui with `FlowWithException` and this don't need any extra calls

Reply:
The UI is not "aware" of the FlowWithException, only a Flow is sent to the UI with an empty list of tasks. The exception message is not send to the UI.

Reply:
In the past, the ui was [aware](https://github.com/kestra-io/kestra/commit/3cc7ef444dec633c53843299b61104ecb4794421#diff-41c09578963e6ebd75970adf75c8369b04c1f2dcadb563fe87aa9632e69b781fR41), we need to reintroduce that directly on flow loading